UI/js/login.js

```diff
@@ -0,0 +1,19 @@
+
+$('#login').submit(function(evt) {
```

Comment:
define #login as a constant at the top. Also try the auto indent, I think it might fix some identification issues

Reply:
sure